src/style.css

```
@@ -0,0 +1,102 @@
+.big-form {
+  position: relative;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.small-form {
+  position: relative;
+  margin-bottom: 20px;
+}
+
+.big {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big:hover {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big:focus {
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.big::placeholder {
+  position: relative;
+  left: 61px;
+  top: 0;
+  right: 18.35%;
+  font-family: Avenir, Franklin Gothc Medium, Arial Narrow, Arial, sans-serif;
```

```suggestion
  font-family: Avenir, sans-serif;
```
enough one generic font